### PR TITLE
Add measurements in `qml.from_qiskit`

### PR DESCRIPTION
**Pull request by obliviateandsurrender**
**Context:** Allows adding PennyLane measurements in `qml.from_qiskit`

**Description of the Change:** Added optional `kwargs` to `qml.load` for adding the required support. 

**Benefits:** Users can specify the measurements they require at the end of build `QNode` from a Qiskit's `QuantumCircuit`.

**Possible Drawbacks:** N/A

**Related GitHub Issues:** N/A

### CODE REVIEW

**Comment by obliviateandsurrender** [comment]
[sc-56071]

File: pennylane/io.py [hunk 1]
```diff
@@ -99,12 +106,14 @@ def from_qiskit(quantum_circuit):
 
     Args:
         quantum_circuit (qiskit.QuantumCircuit): a quantum circuit created in qiskit
+        measurements (list[MeasurementProcess]): the list of PennyLane measurements that
+            overrides the terminal measurements that may be present in the imput circuit.
```
Comment (lillian542) [inline pennylane/io.py:110]:
```suggestion
            overrides the terminal measurements that may be present in the input circuit.
```

File: doc/releases/changelog-dev.md [hunk 2]
```diff
@@ -218,6 +218,10 @@
   are being raised unexpectedly.
   [(#5122)](https://github.com/PennyLaneAI/pennylane/pull/5122)
 
+* Users can specify a list of PennyLane `measurements` they would want as terminal measurements
+  when converting a `QuantumCircuit` using `qml.qml.from_qiskit`.
```
Comment (lillian542) [inline doc/releases/changelog-dev.md:222]:
```suggestion
  when converting a `QuantumCircuit` using `qml.from_qiskit`.
```

File: pennylane/io.py [hunk 3]
```diff
@@ -99,12 +106,14 @@ def from_qiskit(quantum_circuit):
...
 
     Returns:
         function: the PennyLane template created based on the QuantumCircuit
```
Comment (lillian542) [inline pennylane/io.py:113]: Can you add a **Details** section below that shows an example using the `measurements` argument?

**Review by lillian542 — COMMENTED** [review]
Left a couple of docs suggestions! Otherwise I think this is just missing a unit test confirming that `measurements` are passed through to `load`.

File: pennylane/io.py (hunk 3 above)
Comment (obliviateandsurrender) [reply pennylane/io.py]: Added!

**Review by lillian542 — APPROVED** [review]
Looks good to me 🚀

**Review by albi3ro — APPROVED** [review]
Looks great 🎉 

Wishing we had a better way to test it, but I guess this works decently enough until we add a qiskit test runner.

**Pull request merged**